test/JDBC/expected/temp_table_rollback_subxact-vu-prepare.out

```diff
@@ -0,0 +1,70 @@
+CREATE VIEW enr_view AS
+    SELECT
+        CASE
+            WHEN relname LIKE '#pg_toast%' AND relname LIKE '%index%' THEN '#pg_toast_#oid_masked#_index'
+            WHEN relname LIKE '#pg_toast%' THEN '#pg_toast_#oid_masked#'
+            ELSE relname
+        END AS relname
+    FROM sys.babelfish_get_enr_list()
+GO
+
+
+
+
+
+CREATE PROCEDURE test_nested_rollback_in_proc AS
```

Comment:
Can we also have a scenario where we rollback multiple level of savepoints? 

eg:
```
save s1
  save s2
     save s3
rollback s1 -- skipping s2 and s3. 
```